app/gui/controller/engine-protocol/src/binary/serialization.rs

```diff
@@ -531,12 +531,12 @@ impl<'a> DeserializableUnionField<'a, OutboundMessage<'a>> for FromServerPayload
                 })
             }
             OutboundPayload::SUCCESS => Ok(FromServerPayloadOwned::Success {}),
-            OutboundPayload::VISUALISATION_UPDATE => {
-                let payload = message.payload_as_visualisation_update().unwrap();
-                let context = payload.visualisationContext();
+            OutboundPayload::VISUALIzATION_UPDATE => {
+                let payload = message.payload_as_visualization_update().unwrap();
+                let context = payload.visualizationContext();
                 Ok(FromServerPayloadOwned::VisualizationUpdate {
```

Comment:
This does not look good. The casing is all messed up.

The regex used to do this replace should properly inherit the case of the source letter.